test.sh

```diff
@@ -16,6 +16,32 @@ function main(){
     else
         run_production_tests;
     fi
+
+    # TODO: currently only work on development settings
+    if [[ $(ls -l docker-compose.yml) == *"development.yml"* ]]; then
+        if [ $RUN_E2E ]; then
+            if [ $RUN_E2E = 'true' ]; then
```

Comment:
Haven't found a way for lazy evaluation - it is bash, not python, which should probably be changed at some point anyway, but if you know how lazy evaluation in bash works, let me know!